src/tableutils.js

```diff
@@ -219,13 +219,16 @@ export default class TableUtils extends Plugin {
 
 			const tableWalker = new TableWalker( table, { column: insertAt, includeSpanned: true } );
 
-			for ( const { row, column, cell, colspan, rowspan, cellIndex } of tableWalker ) {
+			for ( const { row, cell, cellIndex } of tableWalker ) {
 				// When iterating over column the table walker outputs either:
 				// - cells at given column index (cell "e" from method docs),
 				// - spanned columns (spanned cell from row between cells "g" and "h" - spanned by "e", only if `includeSpanned: true`),
 				// - or a cell from the same row which spans over this column (cell "a").
 
-				if ( column !== insertAt ) {
+				const rowspan = parseInt( cell.getAttribute( 'rowspan' ) || 1 );
```

Comment:
why not take it from table walker if already available?

Reply:
The story of this change is as follows:

1. Before any changes in table repository, it worked. (For some reason :P).
2. After I did some changes, there were bugs with inserting columns.
3. I debugged this part and found out that `colspan` and `rowspan` are always `1` when `includeSpanned: true`. That was causing issues. Actually, the docs says that `colspan` and `rowspan` should be `undefined` when using `includeSpanned`.
4. After I ended working on this PR and got a good understanding how the table walker works, I wanted to revert those changes and see if it is fine, but it wasn't.

So, to sum up:

1. I am not sure what exactly made this to fail, but...
2. It was using properties that were incorrectly set anyway.

Reply:
I also think, that maybe during a lot of "clicking" through different insertions with merged cells I might stumbled upon a case that was not in the original test suite and needed to re-work it for that case 🤷‍♂but this is speculation :).